qiskit/qobj/converters/pulse_instruction.py

```diff
@@ -356,8 +356,9 @@ def convert_acquire(self, instruction):
 
         cmd = commands.Acquire(duration, discriminator=discriminator, kernel=kernel)
         schedule = Schedule()
-        schedule |= commands.AcquireInstruction(cmd, qubit_channels, mem_slots,
-                                                register_slots) << t0
+
+        for acquire_channel, mem_slot, reg_slot in zip(acquire_channels, mem_slots, register_slots):
```

Comment:
Do we verify that the number of acquire_channels, mem_slots and register_slots are equal? `zip` will truncate itself to the shortest iterable in the set of input arguments.

Reply:
yes, they all depends on number of qubtis, they will be the same for all of them